### PR TITLE
Update area-System.IO.Pipelines lead

### PR DESCRIPTION
**Pull request by marek-safar**

### CODE REVIEW

File: docs/area-owners.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
...
-| area-System.IO.Pipelines                       | @kevinpi      | @davidfowl @halter73 @jkotalik                      |                                                                                                                                                                                                                                                                                           |
+| area-System.IO.Pipelines                       | @ericstj      | @davidfowl @halter73 @jkotalik                      |                                                                                                                                                                                                                                                                                           |
```
Comment (jeffhandley) [inline docs/area-owners.md:96]: Should this be @ericstj or @danmoseley?

Comment (danmoseley) [reply docs/area-owners.md]: @rafikiassumani-msft perhaps you?

Comment (BrennanConroy) [reply docs/area-owners.md]: Should be @adityamandaleeka, we look at new pipelines issues in runtime triage

Comment (danmoseley) [reply docs/area-owners.md]: sounds good

**Review by jeffhandley — APPROVED** [review]

**Review by adityamandaleeka — APPROVED** [review]

**Pull request merged**